libs/server-sdk/src/data_components/serialization_adapters/json_data_reader.cpp

```diff
@@ -0,0 +1,39 @@
+#include "json_data_reader.hpp"
```

Comment:
This is a file renaming, not new content (except for updating the `ISerializedDataReader` symbol.)